albumentations/augmentations/transforms.py

```diff
@@ -1422,6 +1423,35 @@ def get_transform_init_args_names(self):
         return ()
 
 
+class ToRGB(ImageOnlyTransform):
+    """Convert the input grayscale image to RGB.
+
+    Args:
+        p (float): probability of applying the transform. Default: 1.
+
+    Targets:
+        image
+
+    Image types:
+        uint8, float32
+    """
+
+    def __init__(self, always_apply=True, p=1.0):
+        super(ToRGB, self).__init__(always_apply=always_apply, p=p)
```

Comment:
Do not need to copy `__init__` if it just calls super method without changes. Please, remove it.

Reply:
I changed the default argument values from `always_apply=False, p=0.5` to `always_apply=True, p=1.0`, so you can write `ToRGB()` instead of `ToRGB(p=1)`. Do I still need to remove this?

Reply:
Oh, you are right. Sorry.